packages/vitest-pool-workers/src/pool/config.ts

```diff
@@ -205,6 +209,22 @@ async function parseCustomPoolOptions(
 				{ imagesLocalMode: true }
 			);
 
+		const wrappedBindings = Object.values(workerOptions.wrappedBindings ?? {});
+
+		const hasAI = wrappedBindings.some((binding) => {
```

Comment:
nit
```suggestion
		const hasAIOrVectorizeBindings = wrappedBindings.some((binding) => {
```

Reply:
actually scrap that. I'm wondering...would it make sense to check for both `hasAIBindings` and `hasVectorizeBindings` and construct the message accordingly? If I have only AI bindings, why would I care about what Vectorize bindings do? WDYT?

Reply:
totally a nit tho